products/centreon.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Centreon
+category: server-app
+tags: php-runtime
+permalink: /centreon
+versionCommand: cat /etc/centreon/centreon.conf
+releasePolicyLink: https://docs.centreon.com/docs/releases/lifecycle/
+releaseDateColumn: true
+activeSupportColumn: true
+eolColumn: Security Support
+
+#identifiers:
+#-   repology: zabbix
+#-   purl: pkg:brew/zabbix
+#-   purl: pkg:github/zabbix/zabbix
+
+auto:
+-   git: https://github.com/centreon/centreon.git
+
+# For non-LTS releases :
+# - support(x) = release(x) + 6 months
+# - eol(x) = release(x) + 7 months
+releases:
+-   releaseCycle: "23"
+    releaseDate: 2023-04-01
+    support: true
+    eol: 2025-04-01
+    latest: "23.04"
+    latestReleaseDate: 2023-04-01 
+    link: https://www.centreon.com/centreon-23-04-discover-the-latest-release/
+
+-   releaseCycle: "22"
+    releaseDate: 2022-05-01
+    support: true
+    eol: 2024-10
+    latest: "22.10"
+    latestReleaseDate: 2022-10
+    link: https://www.centreon.com/centreon-fall22-whats-new-in-the-22-10-software-version/
```

Comment:
Some release notes links can be found on https://download.centreon.com/.

Reply:
Thanks for pointing this, I fix them ⚡